pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -368,7 +400,7 @@ func (k *Kubernetes) ConfigVolumes(name string, service kobject.ServiceConfig) (
 		// For PVC we will also create a PVC object and add to list
 		var volsource *api.VolumeSource
 		if useEmptyVolumes {
-			volsource = k.ConfigEmptyVolumeSource()
+			volsource = k.ConfigEmptyVolumeSource("volume")
```

Comment:
we can't implied here, because volsource is used in `if` and `else` and outside of that too